lib/livebook/runtime/evaluator.ex

```diff
@@ -454,6 +455,32 @@ defmodule Livebook.Runtime.Evaluator do
   defp code_error?(%CompileError{}), do: true
   defp code_error?(_error), do: false
 
+  defp reorder_binding(binding, prev_binding) do
+    # We keep the order of existing binding entries and move
+    # the new ones to the beginning, ordered alphabetically
+
+    binding_map = Map.new(binding)
+
+    prev_idx =
+      prev_binding
+      |> Enum.filter(fn {key, prev_val} ->
+        val = binding_map[key]
+        same?(val, prev_val)
+      end)
+      |> Enum.with_index()
+      |> Map.new(fn {{key, _val}, idx} -> {key, idx} end)
+
+    Enum.sort_by(binding, fn {key, _val} -> {prev_idx[key] || -1, key} end)
+  end
+
+  defp same?(left, right) do
+    if Code.ensure_loaded?(:erts_debug) and function_exported?(:erts_debug, :same, 2) do
```

Comment:
Hrm... why do we need this check? AFAIK it is always available.

Reply:
There is nothing to worry about until there is something to worry about. :D  But yeah, I would skip it.